misc/docs/source/packages/obspy.geodetics.rst

```diff
@@ -10,8 +10,10 @@
        :nosignatures:
 
        ~base.calc_vincenty_inverse
+       ~base.degrees2kilometers
        ~base.gps2dist_azimuth
-       ~base.kilometer2degrees
+       ~base.inside_geobounds
+       ~base.kilometers2degrees
```

Comment:
@megies why 
https://docs.obspy.org/packages/autogen/obspy.geodetics.base.kilometer2degrees.html 
and 
https://docs.obspy.org/packages/autogen/obspy.geodetics.base.kilometers2degrees.html 
both exist ?

Reply:
No idea must be a typo??